library/std/src/collections/hash/map.rs

```diff
@@ -972,6 +972,8 @@ where
     /// map.insert("c", 3);
     ///
     /// let vec: Vec<&str> = map.into_keys().collect();
+    /// vec.sort_unstable();
```

Comment:
A comment here to drive home the point that the `Vec` is in arbitrary order before the sort would be nice

Reply:
Done this in [commit](https://github.com/rust-lang/rust/pull/87598/commits/268a4f0bdbe25d8c47023c0604b0703095d65da9).